.github/workflows/docker-image.yml

```diff
@@ -53,4 +52,44 @@ jobs:
     - name: Logout of Amazon ECR
       if: always()
       run: docker logout ${{ steps.login-ecr.outputs.registry }}
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Deploy to elia_staging
+      uses: peterkimzz/aws-ssm-send-command@master
+      id: ssm
+      with:
+        aws-region: eu-central-1
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        instance-ids: ${{ secrets.INSTANCE_ID }}
+
+        working-directory: /home/ubuntu/docker-stack/
+        command: ls -la
+        command: echo $USER
+        
+        #Setup aws Credentials/Variables
+        command: export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
+        command: export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        command: export AWS_DEFAULT_REGION=eu-central-1
+        command: export TAG=$IMAGE_TAG
+        command: export CPO_PRIVATE_KEY=${{ secrets.CPO_PRIVATE_KEY }}
```

Comment:
Please use .env for file
```
- touch .env
- CPO_PRIVATE_KEY=${{ secrets.CPO_PRIVATE_KEY }} >> .env
- CPO_TOKEN_A=${{ secrets.CPO_TOKEN_A }} >> .env
```

And add in docker-compose:
```
  env_file:
    - .env
```

Reply:
Done